ui/src/dataLoaders/components/side_bar/SideBarTab.tsx

```diff
@@ -20,8 +20,9 @@ class SideBarTab extends Component<Props> {
 
     return (
       <div className={this.className} onClick={this.handleClick}>
-        {this.icon}
-        {label}
+        <pre>
```

Comment:
what problem is this `pre` solving?

Reply:
Without the `pre` there is no space between the icon and the text. We could add the space with styling. But i think it's simpler just to use a `pre` and have the space _be_ text.